beps/0001-notifications-system/README.md

```diff
@@ -85,12 +85,43 @@ The role of the notifications plugin is to manage the lifecycle of notifications
 The notification backend stores notification using the [database service](https://backstage.io/docs/backend-system/core-services/index#database). In particular it needs to store the following information for each notification:
 
 - ID
-- Payload
+- Title
+- Priority
+- Origin
+- Topic (optional)
+- Creation timestamp (generated)
 - Receivers
 - Read status by user
+- Payload
+  - icon (optional)
+  - message (optional)
+  - links (list, optional)
 
 The receivers is **not** a list of users, but rather a filter that describes who should receive the notification. It must be possible to evaluate this filter in a database query, so that we can efficiently fetch all notifications for a given user. The same filter will also be used by the signal backend to determine which users should receive a signal.
 
+The title is mandatory human readable text shortly describing the notifications.
+
+The priority is one of the predefined values to help with presentation of the notifications or their filtering. Values: Critical, High, Normal, Low.
+
+The origin is a string identifying the creator of the notification, i.e. the BE plugin or external system.
```

Comment:
I think this is a good idea and something we need to figure out the exact format of. We'll likely need something like this in [BEP-0003](https://github.com/backstage/backstage/tree/master/beps/0003-auth-architecture-evolution) as well. Ideally there's a service identity that we can carry over directly from the incoming credentials.